packages/rpc-subscriptions-spec/src/__tests__/rpc-subscription-test.ts

```diff
@@ -263,12 +266,14 @@ describe('JSON-RPC 2.0 Subscriptions', () => {
         expect.assertions(1);
         iterable.mockImplementation(async function* () {
             yield Promise.resolve({
-                error: { code: 123, message: 'o no' },
+                error: { code: SOLANA_ERROR__SUBTLE_CRYPTO__DIGEST_UNIMPLEMENTED, message: 'o no block height' },
                 id: 0,
             });
         });
         const subscribePromise = rpc.thingNotifications().subscribe({ abortSignal: new AbortController().signal });
-        await expect(subscribePromise).rejects.toThrow(new SolanaError(123 as SolanaErrorCode, undefined));
+        await expect(subscribePromise).rejects.toThrow(
+            new SolanaError(SOLANA_ERROR__SUBTLE_CRYPTO__DIGEST_UNIMPLEMENTED),
+        );
```

Comment:
CI error seems to be related to these changes - are they intended to be in this PR at all?

Reply:
Hmm. This change was supposed to fix a test failure owing to the fact that error 123 doesn’t exist.

Reply:
Oh, it’s lint I left behind. 

Maybe I’ll try to do this properly with mocks this time.